src/myfunc.cc

```diff
@@ -8,7 +8,8 @@
 void
 myfunc3(void)
 {
```

Comment:
1st function call argument is an uninitialized value [clang-analyzer-core.CallAndMessage]